Homework2_ZhukAlexander/task1.py

```diff
@@ -0,0 +1,12 @@
+# Считает общую цену. Вводится: М-рубли, N-копейки
+# L-количество товара
+
+M = int(input("Введите цену в рублях: "))
+N = int(input("Введите цену в копейках: "))
+L = int(input("Количество предметов: "))
+rub = M * L
+cent = N * L
+if cent > 100:                      # если больше 100 копеек
```

Comment:
redundant comment

Reply:
> 1. Please remove tasks from first homework
> 2. Please rename Pull Request - it contradicts to our guidelines
> 3. It seems tasks from external tasks were pretty easy. you can add some more advanced tasks
> 4. please remove tasks from lessons
> 5. Please address next issues:

put in order the repository. Homework2 without tasks HW1 and lessons
renamed

Reply:
done